e2e/suite1/connections-actions.test.ts

```diff
@@ -0,0 +1,295 @@
+import * as connections from '../helpers/connection-actions';
```

Comment:
Please, rename this file to `relationships-actions.test.ts`

Reply:
Done.